Mathlib/CategoryTheory/PathCategory.lean

```diff
@@ -1,11 +1,12 @@
 /-
 Copyright (c) 2021 Kim Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Kim Morrison
+Authors: Kim Morrison, Robin Carlier
 -/
 import Mathlib.CategoryTheory.EqToHom
 import Mathlib.CategoryTheory.Quotient
 import Mathlib.Combinatorics.Quiver.Path
+import Mathlib.CategoryTheory.MorphismProperty.Basic
```

Comment:
As this increases the imports, could you do the following:
- rename `CategoryTheory.PathCategory` as `CategoryTheory.PathCategory.Basic`;
- add lemmas
```lean
lemma _root_.CategoryTheory.Paths.morphismProperty_eq_top
    (P : MorphismProperty (Paths V))
    (id : ∀ {v : V}, P (𝟙 (of.obj v)))
    (comp : ∀ {u v w : V} (p : of.obj u ⟶ of.obj v) (q : v ⟶ w), P p → P (p ≫ of.map q)) :
    P = ⊤ := sorry`
```
in a new file `CategoryTheory.PathCategory.MorphismProperty`
- in the file `PathCategory.Basic`, remove the import to `MorphismProperty.Basic` and revert to your original formulation of the induction principles (I am sorry I did not anticipate that).